static/js/apps/nl_interface/nl_commentary.tsx

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Commentary for NL results.
+ */
+
+import React from "react";
+
+import { SearchResult } from "../../types/app/nl_interface_types";
+
+interface NLCommentaryPropType {
+  chartsData: SearchResult;
+  hideCharts: boolean;
+  setHideCharts: (v: boolean) => void;
+}
+
+export function shouldHideCharts(respData: any): boolean {
```

Comment:
should this function reside in a util module?

Reply:
The logic is very specific to what's going on in this file, so keeping it here instead of a general util file?